src/ga.js

```diff
@@ -84,7 +84,11 @@ class GeneticAlgorithm {
 
     // Create New Set of humans
     const newGeneration = [];
-    for (let i = 0; i < config.populationSize; i++) {
+    
+    //Elitism: add the clone of the best human to the new generation without mutations for faster neural network learning
```

Comment:
Thank you. I didn't know about Elitism.

Did you run some simulations with this added ? Any progress ?

Reply:
I did. It surely helps the evolution not to get bogged in random mutations and move steadily towards the goal.

However, the introduction of motor noise makes even the best human from the previous run perform poorly on the next one and the evolution loses all the valuable mutations as he gets eliminated.

Have you tried reducing the motor noise?

Reply:
> Have you tried reducing the motor noise?

I haven't. Will give it a try